src/http/client/http_curl.h

```diff
@@ -5,7 +5,12 @@
 #ifndef __HTPP_CURL_INCLUDE__
 #define __HTPP_CURL_INCLUDE__
 
-#include <boost/config.hpp>
+#ifdef _WIN32
```

Comment:
This solution has one drawback, as I understand it won't work in case of C+11 on Linux. Am I right ?

Reply:
Yes, but at the moment there is no C++11 support on Linux, so I decided that it is a good solution for now